src/System.Private.ServiceModel/tests/Scenarios/Contract/Data/DataContractTests.cs

```diff
@@ -94,7 +94,6 @@ public static void NetTcpBinding_DuplexCallback_ReturnsDataContractComplexType()
     // Verify that a callback contract correctly returns a complex type using Xml instead of DataContract when this type is not part of the contract with the ServiceContract attribute.
     [Fact]
     [OuterLoop]
-    [ActiveIssue(292)]
     public static void NetTcpBinding_DuplexCallback_ReturnsXmlComplexType()
```

Comment:
I suspect this test might fail in NET Native for different reasons, but don't let prevent the merge.  We can open a separate issue if that happens.